lib/aws/sns/uri.rb

```diff
@@ -0,0 +1,68 @@
+require "aws-sdk-sns"
+require "addressable/uri"
+
+module Aws
```

Comment:
Should this be in "Aws::SNS" namespace? I got a bit confused earlier and was looking for "URI" class in Amazon docs: https://docs.aws.amazon.com/sdkforruby/api/Aws/SNS.html

Reply:
I think it's fine, it's sort of an amendment to the gem. `show-source` in pry helps when it comes to find where it's defined at runtime.